### ISSUE TITLE
Implement notEqual

### ISSUE THREAD

**Issue by BruceDai** [webmachinelearning/webnn-baseline#111]
Link to https://github.com/webmachinelearning/webnn/issues/375#issuecomment-2292466613

![image](https://github.com/user-attachments/assets/70a98bf9-3e13-4bfc-adf2-7622b1e08b92)

```idl
MLOperand notEqual(MLOperand a, MLOperand b, optional MLOperatorOptions options = {});
```

**Comment by BruceDai** [comment]
Link to PR #113.

**Comment by BruceDai** [comment]
Close it by merged #118 which was re-created from last #113.
